project/apps/api-gateway/src/user/user.controller.ts

```diff
@@ -0,0 +1,132 @@
+  
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpStatus,
+  Param,
+  Patch,
+  Put,
+  Req,
+  Res,
+  Inject,
+} from '@nestjs/common';
+import { ClientProxy } from '@nestjs/microservices';
+import {
+  UpdateUserDto,
+  updateUserSchema
+} from '@repo/dtos/user';
+import { ZodValidationPipe } from '@repo/pipes/zod-validation-pipe.pipe';
+import { Request, Response } from 'express';
+import { firstValueFrom } from 'rxjs';
+@Controller('user')
+export class UserController {
```

Comment:
the `UserController` needs to be set up in `project/apps/api-gateway/src/api-gateway.module.ts`